pkg/test/cluster.go

```diff
@@ -43,7 +43,7 @@ func NewToolchainClusterWithEndpoint(name, secName, apiEndpoint string, status t
 		},
 		ObjectMeta: v1.ObjectMeta{
 			Name:      name,
-			Namespace: "test-namespace",
+			Namespace: ons,
```

Comment:
this one as well - here you can see the usage - it's not set as a label, but as the location namespace of the ToolchainCluster CR

Reply:
done